src/scss/components/_breadcrumbs.scss

```diff
@@ -56,3 +58,27 @@ $breadcrumb-font-weight: $font-weight-bold;
         color: $white;
     }
 }
+
+.c-breadcrumb--compact {
+    @include media('<mid') {
+        .c-breadcrumb-item {
+            display: none;
+        }
+        // only show previous page's link
+        .c-breadcrumb-item:nth-last-child(2) {
+            display: list-item;
+            background-color: rgba($black, 0.2);
```

Comment:
Would be better to include this colour as a variable at the top of this file `$breadcrumb--compact-background: …`